src/mame/bmc/koftball.cpp

```diff
@@ -238,8 +212,10 @@ uint32_t koftball_state::screen_update(screen_device &screen, bitmap_ind16 &bitm
 	girl select after coin up   prev screen prev screen over        under       0x13        0x3a
 	*/
 
+	draw_pixlayer(*m_pixbitmap, cliprect);
+
 	if (BIT(m_priority, 3))
-		copyscrollbitmap_trans(bitmap, *m_pixbitmap, 0, 0, 0, 0, cliprect, 0);
+		copyscrollbitmap(bitmap, *m_pixbitmap, 0, 0, 0, 0, cliprect);
 
 	if (BIT(m_gfx_ctrl, 5)) // TODO: or bit 1?
```

Comment:
Note: I didn't put a great effort in checking how games behave, but this is concealed by the lack of transparent pens in tilemaps 1 & 3.